src/modules/vision_target_estimator/Orientation/VTEOrientation.cpp

```diff
@@ -0,0 +1,345 @@
+/****************************************************************************
+ *
+ *   Copyright (c) 2013-2018 PX4 Development Team. All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright
+ *    notice, this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in
+ *    the documentation and/or other materials provided with the
+ *    distribution.
+ * 3. Neither the name PX4 nor the names of its contributors may be
+ *    used to endorse or promote products derived from this software
+ *    without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+ * FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+ * COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+ * BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS
+ * OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED
+ * AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ ****************************************************************************/
+
+/**
+ * @file VTEOrientation.cpp
+ * @brief Estimate the orientation of a target by processessing and fusing sensor data in a Kalman Filter.
+ *
+ * @author Jonas Perolini <jonspero@me.com>
+ *
+ */
+
+#include <px4_platform_common/px4_config.h>
+#include <px4_platform_common/defines.h>
+#include <drivers/drv_hrt.h>
+
+#include "VTEOrientation.h"
+
+#define SEC2USEC 1000000.0f
+
+namespace vision_target_estimator
+{
+
+using namespace matrix;
+
+VTEOrientation::VTEOrientation() :
+	ModuleParams(nullptr)
+{
+	_vte_aid_ev_yaw_pub.advertise();
+	_targetOrientationPub.advertise();
+
+	_check_params(true);
+}
+
+VTEOrientation::~VTEOrientation()
+{
+	delete _target_estimator_orientation;
+}
+
+bool VTEOrientation::init()
+{
+
+	_target_mode = (TargetMode)_param_vte_mode.get();
+	_vte_TIMEOUT_US = (uint32_t)(_param_vte_btout.get() * SEC2USEC);
```

Comment:
```suggestion
	_vte_TIMEOUT_US = (uint32_t)(_param_vte_btout.get() * 1_s);
```
and declare `using namespace time_literals;` on top to use that `1_s`

Reply:
done in https://github.com/PX4/PX4-Autopilot/pull/22008/commits/6d9b63f087e02b316105704712cdc7b5bdf03d8e